docs/IR.md

```diff
@@ -190,6 +190,7 @@ A serialized function (a _FunctionProto_) has the following properties:
 |---|---|---|
 name|string|The name of the function
 domain|string|The domain to which this function belongs
+overload|string|Part of unique id of function (added in IR version 9)
```

Comment:
is it correct to understand overload is an optional string that maps to "" when unprovided?

You probably also need to update `Nodes` section with the new field.

Reply:
Correct re. first point. Good catch re. second point, will add it.